src/Modifier/AbstractPriceModifier.php

```diff
@@ -1,4 +1,8 @@
 <?php
+namespace Blesta\Pricing\Modifier;
+
+use Blesta\Pricing\Description\AbstractPriceDescription;
+use Blesta\Pricing\Modifier\PriceModifierInterface;
```

Comment:
Don't need to declare `use Blesta\Pricing\Modifier\PriceModifierInterface;`. Since `PriceModifierInterface` is in the same namespace as the one we're in, it's available by default.